<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>424</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuSpace</string>
  </property>
  <property name="windowIcon">
   <iconset resource="quspace.qrc">
    <normaloff>:/uspace_large</normaloff>:/uspace_large</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>100</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="leftPlaceHolder">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QPushButton" name="backButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>7</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>Back</string>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 0px;
background: rgb(255,255,255,0);
image: url(:/back_icon);
/**
QPushButton {
border: 1px solid #8f8f91;
border-radius: 3px;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             					     stop: 0 #f6f7fa,
                             					     stop: 0.3 #dadbde,
                             					     stop: 0.7 #dadbde,
                                                     stop: 1 #f6f7fa);
}
QPushButton::hover {
border: 1px solid #8f8f91;
border-radius: 3px;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
													 stop:0 #dadbde,
                             					     stop: 0.3 #f6f7fa,
                             					     stop: 0.7 #f6f7fa,
                                                     stop:1 #dadbde);
}
QPushButton:pressed {
border: 2px solid navy;
border-radius: 3px;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
													 stop:0 #dadbde,
                             					     stop: 0.3 #f6f7fa,
                             					     stop: 0.7 #f6f7fa,
                                                     stop:1 #dadbde);
}
**/</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>100</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="toolsButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>7</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>Tools</string>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 0px;
background: rgb(255,255,255,0);
image: url(:/tools_icon);
/**
QPushButton {
border: 1px solid #8f8f91;
border-radius: 3px;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             					     stop: 0 #f6f7fa,
                             					     stop: 0.3 #dadbde,
                             					     stop: 0.7 #dadbde,
                                                     stop: 1 #f6f7fa);
}
QPushButton::hover {
border: 1px solid #8f8f91;
border-radius: 3px;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
													 stop:0 #dadbde,
                             					     stop: 0.3 #f6f7fa,
                             					     stop: 0.7 #f6f7fa,
                                                     stop:1 #dadbde);
}
QPushButton:pressed {
border: 2px solid navy;
border-radius: 3px;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
													 stop:0 #dadbde,
                             					     stop: 0.3 #f6f7fa,
                             					     stop: 0.7 #f6f7fa,
                                                     stop:1 #dadbde);
}
**/
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="taskbarButton">
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>Taskbar</string>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 0px;
background: rgb(255,255,255,0);
image: url(:/taskbar_icon_invisible);
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="quitButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>7</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>Exit</string>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 0px;
background: rgb(255,255,255,0);
image: url(:/exit_icon);
/**
QPushButton {
border: 1px solid #8f8f91;
border-radius: 3px;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             					     stop: 0 #f6f7fa,
                             					     stop: 0.3 #dadbde,
                             					     stop: 0.7 #dadbde,
                                                     stop: 1 #f6f7fa);
}
QPushButton::hover {
border: 1px solid #8f8f91;
border-radius: 3px;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
													 stop:0 #dadbde,
                             					     stop: 0.3 #f6f7fa,
                             					     stop: 0.7 #f6f7fa,
                                                     stop:1 #dadbde);
}
QPushButton:pressed {
border: 2px solid navy;
border-radius: 3px;
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
													 stop:0 #dadbde,
                             					     stop: 0.3 #f6f7fa,
                             					     stop: 0.7 #f6f7fa,
                                                     stop:1 #dadbde);
}
**/</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>quitButton</tabstop>
 </tabstops>
 <resources>
  <include location="quspace.qrc"/>
 </resources>
 <connections/>
</ui>
